### PR TITLE
Gimbap fix ie9

### PR DESCRIPTION
**Pull request by uplne**
Life would be so much happier without IE.

IE9:
screen shot 2016-03-23 at 11 35 10
screen shot 2016-03-23 at 11 59 45

@regiskuckaertz

### CODE REVIEW

File: static/src/stylesheets/module/commercial/gimbap/_gimbap.scss [hunk 1]
```diff
@@ -333,17 +340,29 @@ $gimbap-jobs-reversed: #42bfda;
...
 
 .gimbap-wrap__body--2x1x1 > .gimbap:first-child,
 .gimbap-wrap__body--1x2x1 > .gimbap:nth-child(2),
 .gimbap-wrap__body--1x1x2 > .gimbap:nth-child(3) {
     @include mq(tablet) {
-        flex: 0 0 calc(66.67% - 20px);
+        width: calc(66.67% - 20px);
+
+        .has-flex & {
+            flex: 2;
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/commercial/gimbap/_gimbap.scss]: Oooh, how could I not see this one! Very good, I'm stealing it :-)

**Comment by regiskuckaertz** [comment]
Seems good, although I would have used `display: table` personally, instead of floats.

**Comment by regiskuckaertz** [comment]
:+1:

**Comment by uplne** [comment]
margin doesn't work on `display: table` so as I am not afraid of `float` I went for it.

**Pull request merged**